internals/generators/container/stateless.js.hbs

```diff
@@ -46,7 +55,7 @@ function {{ properCase name }}() {
 
 {{#if wantActionsAndReducer}}
 const mapStateToProps = createStructuredSelector({
-  {{name}}: makeSelect{{properCase name}}(),
+  {{ lowerCase name }}: makeSelect{{properCase name}}(),
```

Comment:
`createStructuredSelector` is just a helper that you can use to transform:

```js
const mapStateToProps = (state) => ({
  x: selectorA(state),
  y: selectorB(state),
})
```

to

```js
const mapStateToProps = createStructuredSelector({
  x: selectorA,
  y: selectorB,
})
```

Not everyone uses it, so maybe we should start with something simpler, anyhow, we won't satisfy everyone's needs, devs has each their own preferences, coding styles and techniques. I'd rather discuss the necessity of a memorized selector factory (`make...`) it's a technique that's useful only in some specific set of cases.